src/mars.d

```diff
@@ -81,10 +81,12 @@ private  void usage()
     static if (TARGET_WINDOS)
     {
         const(char)* m32mscoff = "\n  -m32mscoff       generate 32 bit code and write MS-COFF object files";
+        const(char)* mscrtlib  = "\n  -mscrtlib=<name> MS C runtime library to reference from main/WinMain/DllMain";
     }
     else
     {
         const(char)* m32mscoff = "";
+        const(char)* mscrtlib  = "";
```

Comment:
You are looking at the usage message here. The default is set after command line parsing below: https://github.com/dlang/dmd/pull/6244/files#diff-2a089032228a74524c0933f9bd7e71deR1027